### ISSUE TITLE
Function "DataLoader()" does not exist! 

### ISSUE THREAD

**Issue by nuclear718** [FluxML/Flux.jl#1109]
**There is a function "DataLoader()" does not exist!** 
(The below codes were almost fully copy from official DOC.>Training>Dataset )

`using CSV, DataFrames, DataFramesMeta, Plots,Dates,Statistics,GLM,Flux`
 `r = rand(28, 28, 60000)`
 `t = rand(0:9, 60000) `
 `data = DataLoader(r, t, batchsize=128) `
**return:**  
UndefVarError: DataLoader not defined
Stacktrace:
 [1] top-level scope at In[34]:3

or, slightly modify the function name to full long name:
`using CSV, DataFrames, DataFramesMeta, Plots,Dates,Statistics,GLM,Flux`
`r = rand(28, 28, 60000)`
`t = rand(0:9, 60000)`
`data = Flux.Data.DataLoader(r, t, batchsize=128) `

**still return:** 
UndefVarError: DataLoader not defined
Stacktrace:
 [1] getproperty(::Module, ::Symbol) at .\Base.jl:13
 [2] top-level scope at In[35]:3

My ENV. detail information:
Julia Version 1.3.1
Commit 2d5741174c (2019-12-30 21:36 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: AMD Ryzen 5 3600X 6-Core Processor             
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, znver1)

My Pkg status:
[fbb218c0] BSON v0.2.5
  [336ed68f] CSV v0.6.0
  [3895d2a7] CUDAapi v4.0.0
  [c5f51814] CUDAdrv v6.2.2
  [be33ccc6] CUDAnative v3.0.2
  [3a865a2d] CuArrays v2.0.1
  [1b08a953] Dash v0.1.0 #master (https://github.com/plotly/Dash.jl.git)
  [a93c6f00] DataFrames v0.20.2
  [1313f7d8] DataFramesMeta v0.5.0
  [587475ba] Flux v0.8.3
  [38e38edf] GLM v1.3.9
  [cd3eb016] HTTP v0.8.13
  [09f84164] HypothesisTests v0.9.2
  [7073ff75] IJulia v1.21.1
  [add582a8] MLJ v0.10.3
  [91a5bcdd] Plots v0.29.9

**Comment by AdarshKumar712** [comment]
@nuclear718 Your Flux version is 0.8.3. DataLoader is not available in that version. Try installing the current stable version 0.10.3 for flux using `] add Flux@0.10.3`

**Comment by nuclear718** [comment]
> @nuclear718 Your Flux version is 0.8.3. DataLoader is not available in that version. Try installing the current stable version 0.10.3 for flux using `] add Flux@0.10.3`

Sorry ! 

Thanks for your reply.

**Comment by cecroc** [comment]
I ran into the same error with v0.10.3 

Using the same example as above: 

```
using Flux
r = rand(28, 28, 60000)
t = rand(0:9, 60000)
data = DataLoader(r, t, batchsize=128)
```

Gives the same error: "UndefVarError: DataLoader not defined"

I only have Flux in my working environment at the moment: 

pkg> st
    Status `C:\Users\...\Project.toml`
  [587475ba] Flux v0.10.3

I get the same error if I instead write "Flux.Data.DataLoader" 

Thanks in advance for any insight!

**Comment by CarloLucibello** [comment]
DataLoader is defined in the Data module, as documented in the docs. Try
``` julia
julia> using Flux.Data: DataLoader
```

**Comment by cecroc** [comment]
Thanks for the quick reply! 

Long story short, it works now. In case it helps anyone else: 

I noticed that 
`using Flux.Data: DataLoader`
Gives the same error: "UndefVarError: DataLoader not defined"
But that 
`using Flux.Data: MNIST, Iris, FashionMNIST, CMUDict, cmudict `
works just fine (the last export in the Data.jl file (Housing) also gave an UndefVarError).

My best guess is it was somehow finding the old version of Flux in my packages (still mysterious to me how this is possible when `]st` only listed v0.10.3...). I removed Flux with the package manager, deleted the downloaded Flux files, closed and re-opened Julia, reinstalled Flux, and now it works fine.
